docs/tutorials/tutorial-update-data.md

```diff
@@ -23,161 +23,236 @@ sidebar_label: Update existing data
   ~ under the License.
   -->
 
+:::info
```

Comment:
Suggest removing the admonition. The "This tutorial" statement says it uses SQL and all the examples are SQL.